.github/workflows/lambda_scribe_proxy.yml

```
@@ -39,3 +39,4 @@ jobs:
           function_name: gh-ci-scribe-proxy
           source: aws/lambda/scribe-proxy/lambda_function.py
           timeout: 60 # 60 seconds as the max duration of the lambda_function
+          memory_size: 512
```

Nit, please add a comment. Also, wouldn't 256Mb(or even say 192Mb) be a better move?
```suggestion
          memory_size: 512 # 512 Mb
```

---

agreed, let's try 256 MB first.